uaclient/util.py

```diff
@@ -352,11 +352,18 @@ def get_platform_info() -> "Dict[str, str]":
 @lru_cache(maxsize=None)
 def is_container(run_path: str = "/run") -> bool:
     """Checks to see if this code running in a container of some sort"""
```

Comment:
This new chroot special case might be a little confusing to future readers. Consider adding a note to the docstring here along the lines of: "chroots can appear to be containers but ua shouldn't treat them as containers, so we explicitly check for chroot and return False if we're in one"

Reply:
Yes, that's an excellent idea